Sources/CLI/BuildCommand.swift

```diff
@@ -169,7 +189,18 @@ extension Application {
                     throw ValidationError("builder is not running")
                 }
 
-                let dockerfile = try Data(contentsOf: URL(filePath: file))
+                let buildFile: Path
+                if let explicitPath = file {
+                    buildFile = Path(explicitPath)
+                    guard buildFile.exists else {
+                        throw ValidationError("Provided container build file does not exist: \(buildFile.string)")
+                    }
+                } else {
+                    buildFile = try BuildCommand.checkForDockerOrContainerfile(in: buildDirectory)
+                }
```

Comment:
This could be nicer if we just check to see if the user has explicitly provided a file, if they have then we set `dockerfile` to that, else we use the new function to find the file. 

Maybe all of the logic should go into the new function and then the validation only needs to be done once, regardless of which file is provided or found.